guides/doc-Managing_Hosts/topics/using_insights_with_satellite_hosts.adoc

```diff
@@ -9,7 +9,7 @@ You can sort by category, view details of the impact and resolution, and then de
 To use Red{nbsp}Hat Insights to monitor hosts that you manage with {Project}, you must first install Red{nbsp}Hat Insights on your hosts and register your hosts with Red{nbsp}Hat Insights.
 
 For new {Project} hosts, you can install and configure {Project} hosts with Insights during registration using the global registration template.
-For more information, see {ManagingHostsDocURL}registering-a-host-to-project-using-the-global-registration-template_managing-hosts[Registering a Host to {ProjectName} Using the Global Registration Template] in the _Managing Hosts_ guide.
+For more information, see {ManagingHostsDocURL}registering-a-host_managing-hosts[Registering a Host to {ProjectName} Using the Global Registration Template] in the _Managing Hosts_ guide.
```

Comment:
Wow and this validates because this is rendered only in Satellite context and the ID is still old in the Satellite documentation.